src/main/java/org/asciidoclet/package-info.java

```diff
@@ -33,7 +33,7 @@
  *   <version>2.9</version>
  *   <configuration>
  *     <source>1.7</source>
- *     <doclet>org.asciidoctor.Asciidoclet</doclet>
+ *     <doclet>org.asciidoclet.Asciidoclet</doclet>
```

Comment:
I didn't catch this until just now, but org.asciidoctor is going to be correct. It should stay part of the asciidoctor org.

Reply:
Okay.

Reply:
I will have to move everything into the `org.asciidoctor.asciidoclet` package, though, to avoid package conflicts with asciidoclet itself. The module system enforces this.